src/zenml/cli/utils.py

```diff
@@ -230,14 +230,12 @@ def format_integration_list(
     """
     list_of_dicts = []
     for name, integration_impl in integrations:
```

Comment:
Not sure why we even need these type ignore, I guess it's because the integrations here are typed as "IntegrationMeta" in the function signature. Can you try replacing that by our actual integration class and see if the the ignores are still necessary?

Reply:
Nope. I still get:

```
+ mypy src/zenml
src/zenml/cli/utils.py:233: error: "IntegrationMeta" has no attribute "check_installation"  [attr-defined]
src/zenml/cli/utils.py:238: error: "IntegrationMeta" has no attribute "REQUIREMENTS"  [attr-defined]
```